src/box/lua/console.h

```diff
@@ -31,12 +31,21 @@
  * SUCH DAMAGE.
  */
 
+#include <stdbool.h>
 #include "small/rlist.h"
 
 #if defined(__cplusplus)
 extern "C" {
 #endif /* defined(__cplusplus) */
 
+/**
+ * If a local console is exited and there are active libev events (e.g. there's
+ * a background fiber running), Tarantool seemingly freezes - console stops to
+ * work, typed characters are not echoed. This flag is used by main() to display
+ * a message to stdout to make things clear.
+ */
+extern bool is_console_exited;
```

Comment:
Nit: I think that exposing it using an accessor function would look more accurate. Don't know, whether there are formal rules regarding this point. Feel free to resolve the discussion without changes.